sky/core.py

```diff
@@ -757,3 +757,21 @@ def storage_delete(name: str) -> None:
                                     source=handle.source,
                                     sync_on_reconstruction=False)
         store_object.delete()
+
+
+def storage_create(name: Optional[str] = None,
+                   source: Optional[str] = None,
+                   stores: Tuple[str] = None):
+    """"Create a storage.
+
+    """
+    storage = data.Storage(name, source)
+
+    stores = [] if stores is None else stores
+    stores = [store_type.upper() for store_type in stores]
+
+    # Add stores from tuple to storage object
+    if 'GCS' in stores:
```

Comment:
Is it possible to do this in parallel? (i.e. I dont want to wait for the bucket to first upload to GCS and then S3 if I specify GCS,S3 for stores)

Reply:
Implemented concurrently adding stores by defining a top level helper function in core.py to resolve pickling errors due to passing in a storage object method to p.map.